### PR TITLE
godoc: add page

### PR DESCRIPTION
**Pull request by danieldc**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/godoc.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+> Show documentation for go packages.
+
+- Display help for name (named "Printf" in this case) of package (named "fmt" in this case):
+
+`godoc {{fmt}} {{Printf}}`
+
+- Serve documentation as an HTML web server on http://localhost:6060/":
+
+`godoc -http=:6060`
```
Comment (agnivade) [inline pages/common/godoc.md]: 6060 in tokens (`{{}}`)

File: pages/common/godoc.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# godoc
+
+> Show documentation for go packages.
+
+- Display help for name (named "Printf" in this case) of package (named "fmt" in this case):
+
+`godoc {{fmt}} {{Printf}}`
+
+- Serve documentation as an HTML web server on http://localhost:6060/":
```
[the remark this answers is not in the document]
Comment (pxgamer) [reply pages/common/godoc.md]: Yeah, definitely a bit more readable. 👍 Also, as there wasn't a starting quote on the URL, this looks a bit odd.

**Comment by danieldc** [comment]
Added `godoc {{package}}`, simplified wording for web serving use case and `godoc {{package}} {{name}}`.

(sorry for delay -- I'm traveling and wifi coverage is spotty).

File: pages/common/godoc.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# godoc
+
+> Show documentation for go packages.
+
+- Display help for name (named "Printf" in this case) of package (named "fmt" in this case):
```
[the remark this answers is not in the document]
Comment (danieldc) [reply pages/common/godoc.md]: done.

File: pages/common/godoc.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (danieldc) [reply pages/common/godoc.md]: done.

**Comment by agnivade** [comment]
> About index related commands: would you be OK accepting the PR without them?

Sure, I can add them myself. Or are you suggesting you don't want the index commands in the page ? I think it is a very useful option to anyone seriously using godoc.

**Comment by danieldc** [comment]
>  Sure, I can add them myself. Or are you suggesting you don't want the index commands in the page 

I haven't interacted with enough go users to judge this, and I don't use them myself hence I don't feel that I'm the right person to do them.
If you think they are useful, then I think it will be great to add them.

FWIW, here's what I would have done (side note: I don't mind if you add this to this PR, change the content as you see fit, or do it in a different PR)

> - Serve documentation as a web server on port "6060", and enable word searching (i.e. "full text search"), rather than just indentifiers:
> 
> `godoc -http=:{{6060}} -indexi`
> 
> - Serve documentation as a web server on port "6060", and use the given file as the full text search index. The second command will refresh the index.
> `godoc -http=:{{6060}} -index -index_files=/tmp/godoc.fsix`
> `godoc -goroot=/usr/local/go -index -index_files=/tmp/godoc.fsix -write_index`

**Comment by agnivade** [comment]
Thank you. I will open a separate PR. Since we have space left for other commands, it should be fine to add these.

**Pull request merged**